### PR TITLE
add macos build script, remove Kokoro configs

### PR DESCRIPTION
**Pull request by akerekes**

### CODE REVIEW

File: kokoro/continuous_mac.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands to stderr.
+set -x
+
+curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-154.0.1-darwin-x86_64.tar.gz -o google-cloud-sdk-154.0.1-darwin-x86_64.tar.gz
```
Comment (loosebazooka) [inline kokoro/continuous_mac.sh]: I thought there was a way to get latest?

**Comment by akerekes** [comment]
re: always downloading the sdk: MacOS agents don't have Cloud SDK currently
neither Maven, so I'm adding mvnw to the project in this PR so it's not yet complete.

**Comment by akerekes** [comment]
FYI I haven't updated the travis/apveyor scripts to use mvnw in hope of Kokoro replacing them in the near future.

**Comment by akerekes** [comment]
@loosebazooka @patflynn PTAL

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```
Comment (patflynn) [inline .mvn/wrapper/maven-wrapper.properties]: use https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.zip

File: kokoro/continuous_mac.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Display commands to stderr.
+set -x
+
+curl https://sdk.cloud.google.com | bash
+GOOGLE_CLOUD_SDK_HOME=/Users/kbuilder/google-cloud-sdk
+"$GOOGLE_CLOUD_SDK_HOME"/bin/gcloud components install app-engine-java
+
+# temporary workaround until mvn is available in the image by default
+# the integration tests rely on Maven being installed, cannot use the wrapper
+curl http://www-us.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz -o apache-maven-3.5.0-bin.tar.gz
```
Comment (patflynn) [inline kokoro/continuous_mac.sh]: https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.tar.gz

File: .mvn/wrapper/maven-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.zip
```
Comment (loosebazooka) [inline .mvn/wrapper/maven-wrapper.properties]: does this break if we update the maven wrapper?

**Comment by patflynn** [comment]
you kind of have to manually manage the versions.

**Pull request merged**